apps/passport-server/src/apis/lemonade/graphQLClient.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  ApolloClient,
+  DefaultContext,
+  DocumentNode,
+  InMemoryCache,
+  NormalizedCacheObject
+} from "@apollo/client";
+import urljoin from "url-join";
+import {
+  getEventTicketTypesQuery,
+  getHostingEventsQuery,
+  updateEventCheckinMutation
+} from "./queries";
+import {
+  GetEventTicketTypesResponse,
+  GetHostingEventsResponse,
+  LemonadeCheckin,
+  LemonadeEvents,
+  LemonadeTicketTypes,
+  UpdateEventCheckinResponse
+} from "./types";
+
+/**
+ * Wraps an Apollo GraphQL client. There is one client instance per backend
+ * URL, so in practice there is likely to be only one instance.
+ */
```

Comment:
I've changed this so that we re-instantiate the client on each request. Caching it seemed useful when I thought we would be doing many requests per pipeline load, as we do with Pretix, but now we don't use `LemonadeClient` during pipeline loads at all (since we have the REST CSV endpoint instead), so it does not seem worth it.